modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/connect/ConnectCommand.java

```diff
@@ -39,17 +45,37 @@ public class ConnectCommand extends BaseCommand implements Runnable {
     @Parameters(description = NODE_URL_DESC, descriptionKey = CLUSTER_URL_KEY, converter = UrlConverter.class)
     private URL nodeUrl;
 
+    /** Node name option. */
+    @Option(names = {NODE_NAME_OPTION_SHORT, NODE_NAME_OPTION}, description = NODE_NAME_DESC)
```

Comment:
Maybe we can combine these into the single parameter and detect whether this is a node name or a node URL?